schemas/JSON/manifests/v1.2.0/manifest.installer.1.2.0.json

```diff
@@ -209,7 +231,7 @@
       "type": [ "array", "null" ],
       "items": {
         "type": "string",
-        "pattern": "^[a-z][-a-z0-9\\.\\+]*$",

```

Comment:
>"pattern": "^[a-z][-a-z0-9\\.\\+]*$", [](http://example.com/codeflow?start=8&length=37)

The initial regex is actually copied from appx manifest schema, if Windows supports starting with numbers, then I guess we'll go with this change.

Reply:
Removed regex constraint as it is no longer needed anymore.